src/Groups/GAPGroups.jl

```diff
@@ -1590,6 +1592,83 @@ function _map_word_syllable(vi::Pair{Int, Int}, genimgs::Vector, genimgs_inv::Ve
 end
 
 
+@doc Markdown.doc"""
+    syllables(g::FPGroupElem)
+
+Return the syllables of `g` as a list of pairs `(gen,exp)` where
+`gen` is the index of a generator and `exp` is an exponent. 
+
+# Examples
+```jldoctest
+julia> F = free_group(2);  F1 = gen(F, 1);  F2 = gen(F, 2);
```

Comment:
```suggestion
julia> F = free_group(2);  F1, F2 = gens(F);
```
? Feel free to dismiss